package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.1",
   "description": "A starting point for the CSS Frameworks CA project",
   "scripts": {
-    "test": "echo \"We will learn more about testing in the Workflow course\" && exit 1"
+    "test": "echo \"We will learn more about testing in the Workflow course\" && exit 1",
+    "start": "live server"
```

Comment:
You have no script to transpile scss code to css. Why? To pass this CA, you need to transpile scss to css using sass npm package